### ISSUE TITLE
Cannot display image file

### ISSUE THREAD

**Issue by UgnilJoZ** [ArturKovacs/emulsion#14]
Below is the output when trying a valid PNG image:

```
$ emulsion Animated_PNG_example_bouncing_beach_ball.png 
Error occured while loading image: Format error decoding Png: invalid filter method (250)
... caused by: invalid filter method (250)

Found latest version tag v1.9
Current version is '1.9.0'
Parsed latest version is '1.9.0'
```

To be fair, it is an APNG.

But to my understanding, every PNG viewer should be able to display at least one frame in an APNG image, as it is [backwards compatible](https://en.wikipedia.org/wiki/APNG) to PNG. So my expectation would be to display some or even an animated image.

The image is located [here](https://upload.wikimedia.org/wikipedia/commons/1/14/Animated_PNG_example_bouncing_beach_ball.png) on Wikimedia.

**Comment by tuncer** [comment]
Relevant patch: https://github.com/image-rs/image-png/pull/93

**Comment by tuncer** [comment]
I was mistaken, that patch is for encode.

**Comment by tuncer** [comment]
This could be a [possible issue](https://github.com/image-rs/image-png/blob/596c22c1f4457f1b93d0324e4c452363fb25fa35/src/decoder/mod.rs#L491) in image-png. Also see [FilterType](https://github.com/image-rs/image-png/blob/596c22c1f4457f1b93d0324e4c452363fb25fa35/src/filter.rs#L18).

**Comment by ArturKovacs** [comment]
Fixed by 751e07bd17a380b1835d42001c28bae199ae9f7c
